docs/configuration.md

```diff
@@ -1752,6 +1752,14 @@ showDF(properties, numRows = 200, truncate = FALSE)
     Executable for executing R scripts in client modes for driver. Ignored in cluster modes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.r.shell.command</code></td>
+  <td>R</td>
+  <td>
+    Executable for executing sparkR shell in client modes for driver. Ignored in cluster modes. It is the same as environment variable <code>SPARKR_DRIVER_R</code>, but take precedence over it.
+    <code>spark.r.shell.command</code> is used for interactive mode of sparkR (sparkR shell) while <code>spark.r.driver.command</code> is used for the batch mode (running sparkR script).
```

Comment:
here `(sparkR shell)` means `interactive mode of sparkR` and `(running sparkR script)` means `batch mode`. If I use 'sparkR shell' directly, then I might to rephrase the whole senentence.